everglades_workflow.sh

```diff
@@ -16,9 +16,10 @@ echo "INFO [$(date "+%Y-%m-%d %H:%M:%S")] Loading required modules"
 source /etc/profile.d/modules.sh
 
 ml conda
-conda env create --force --file /blue/ewhite/everglades/EvergladesTools/Zooniverse/environment.yml
```

Comment:
Delete instead of comment out?

Reply:
Yeah, in the docs sounds better. Speaking of which, I was going to note that we should add a description of this setup to the README as part of the PR.